xrpl_types/src/amount.rs

```diff
@@ -20,18 +20,18 @@ impl Default for Amount {
 }
 
 impl Amount {
-    pub fn issued(value: &str, currency: &str, issuer: &str) -> Self {
+    pub fn issued(
+        value: impl Into<String>,
+        currency: impl Into<String>,
+        issuer: impl Into<String>,
+    ) -> Self {
         Self::Issued {
-            value: value.to_string(),
-            currency: currency.to_string(),
-            issuer: issuer.to_string(),
+            value: value.into(),
+            currency: currency.into(),
+            issuer: issuer.into(),
         }
     }
 
-    pub fn iou(value: &str, currency: &str, issuer: &str) -> Self {
```

Comment:
seemed confusing having two methods doing the same but different names, have not seen that in Rust before

Reply:
Yeah, I wanted to keep this as an alias. But I think the term IOU is not used any more on XRPL, so I am good to remove it.